addons/mail/static/src/models/messaging_initializer.js

```diff
@@ -86,7 +86,8 @@ registerModel({
             partner_root,
             publicPartners,
             shortcodes = [],
-            starred_counter = 0
+            starred_counter = 0,
+            isLinkPreviewEnabled = true
```

Comment:
trailing `,`

Reply:
It was not present before my change, but I guess it was a mistake